### ISSUE TITLE
Suggestion: Tutorial

### ISSUE THREAD

**Issue by giannikosk** [audunarn/ANYstructure#132]
Hello, i have been looking for an excel sheet or an app to calculate the strength of a floating structure according to the DNV-GL-C202 guide line for shell buckling and i came across this page. It looks very interesting and its obvious that the team has put good work in this project, however trying to use this app has been a bit challenging for me. Honestly i havent put too much effort in it, and I could try a bit more, but i was thinking that a tutorial would be very helpful on a worked example to get the hang of the tool fast. Could you consider a video tutorial demonstrating the tool?

**Comment by audunarn** [comment]
Hi!

Yes you have a point :) It is on my to do list. Have you loaded the example
and read the documentation?

Audun

tir. 12. nov. 2024 kl. 18:40 skrev giannikosk ***@***.***>:

**Comment by giannikosk** [comment]
Yes i checked the documentation, because the problem i wanted to solve was very simple i thought i could use the API. The main issue i faced was that i would like to have somewhere in the documentation the order the methods need to be called so all prerequisites for calculations are done correctly. In the end i used the GUI which was indeed easy and well documented. Only issue i had with the GUI is the scaling. I have an odd resolution 2880x1800 and i couldnt get the app to scale properly, so parts of the GUI were compressed and unreadable. I manually scaled my resolution to make sense of the input boxes. I see however that is also a resolved issue on the GUI would you like me to open a new issue showcasing the problem or do you consider this resolved from the other issue?

**Comment by audunarn** [comment]
Yes please show me. The resolution scaling may need some modifications.

Also, I know the API need some improvement. However, if you see in the documentation an example is provided on page 4:

https://sites.google.com/view/anystructure/documentation

**Comment by audunarn** [comment]
Things does take some time :) 
https://www.youtube.com/@ANYopenSoft
